src/Microsoft.AspNetCore.Sockets.Client.Http/WebSocketsTransport.cs

```diff
@@ -56,7 +56,7 @@ public WebSocketsTransport(HttpOptions httpOptions, ILoggerFactory loggerFactory
             _logger = (loggerFactory ?? NullLoggerFactory.Instance).CreateLogger<WebSocketsTransport>();
         }
 
-        public async Task StartAsync(Uri url, IDuplexPipe application, TransferMode requestedTransferMode, IConnection connection)
+        public async Task StartAsync(Uri url, IDuplexPipe application, TransferFormat requestedTransferMode, IConnection connection)
```

Comment:
Name again. There are more later in the PR that I'll skip commenting. Doing a search to see what is missed

Reply:
Doing a find-replace